pages/cloud/lake-effect.md

```diff
@@ -1,47 +1,71 @@
+Although high performance computing (HPC) is our largest and most widely used service, CCR recognizes that not all research requires number crunching on large scale computers (clusters). LakeEffect, was put into service to provide flexible infrastructure to support research computing such as webservers, databases, and long-running services that are not suited for HPC clusters. This along with our HPC platform allows us to provide a complete set of tools to support all the different research groups and collaborations througout UB.
 
 
-### Lakeeffect Quick Start
+### LakeEffect Quick Start
 
-1. Details on Pricing and Subscription packages is [here](subscriptions.md)
+1. Details on Pricing and Subscription packages is [here](#subscriptions)
 2. Request access through the [coldfront](../portals/coldfront.md) allocations management portal
 3. Once you have an active allocation, you can access the cloud through the [Dashboard](https://dashboard.cloud.ccr.buffalo.edu)
-4. Documenation on using Lakeeffect Cloud and Openstack:
-	- [Using LakeEffect Cloud](./using.md)
-	- [OpenStack Documentation](https://docs.openstack.org/)
-
+4. Documenation on using LakeEffect Cloud and Openstack:
+	* [Using LakeEffect Cloud](./using.md)
+	* [OpenStack Documentation](https://docs.openstack.org/)
 
 ### Features
 
-Features of Lake Effect:
+Features of LakeEffect:
 
-- Deploy virtual servers in minutes
-- Storage volume creation/snapshot
-- VPC Networking with access to public IP address space as well as the ability to create private networks
-- Wide range of Machine Types available inclunding GPUs and IO Optimized Storage
-- Various Linux operating systems are supported.  _**NOTE**: Windows images are not provided be CCR at this time_
-- Public Images of opensource Linux Distributions available as well as the ability for users to upload their own
-- Load balancing of virtual resources across multiple physical machines
-- Support for Floating IP addresses
-- Easily manage access to VMs using Security Groups and Rules
-- Built on Openstack Cloud Software [Openstack](https://www.openstack.org/) 
-- Fault tolerant components to minimize risk of downtime due to underlying hardware failure
+* Deploy virtual servers in minutes
+* Storage volume creation/snapshot
+* VPC Networking with access to public IP address space as well as the ability to create private networks
+* Wide range of Machine Types available inclunding GPUs and IO Optimized Storage
+* Various Linux operating systems are supported.  _**NOTE**: Windows images are not provided be CCR at this time_
+* Public Images of opensource Linux Distributions available as well as the ability for users to upload their own
+* Load balancing of virtual resources across multiple physical machines
+* Support for Floating IP addresses
+* Easily manage access to VMs using Security Groups and Rules
+* Built on Openstack Cloud Software [Openstack](https://www.openstack.org/) 
```

Comment:
This link is kind of repetitive.  Did you mean to make "Openstack Cloud Software" the link?

Reply:
fixed - made "Openstack Cloud Software" the link